sigar_interface.go

```diff
@@ -15,6 +16,8 @@ type Sigar interface {
 	GetSystemDistribution() (SystemDistribution, error)
 }
 
+var ErrNotImplemented error = errors.New("Collection not implemented")
```

Comment:
Do you want to specifically say "Collection not implemented _for this operating system_"?